CMake/InstallResources.cmake

```diff
@@ -11,33 +11,51 @@
 # Generate and install the config files needed for the samples
 ##################################################################
 
+if (NOT OGRE_MEDIA_PATH)
+  if (WIN32 OR APPLE)
+    set(OGRE_MEDIA_PATH "Media")
+  elseif (UNIX)
+    set(OGRE_MEDIA_PATH "share/OGRE/Media")
+  endif()
+endif()
+
+set(OGRE_MEDIA_DIR_REL "${CMAKE_INSTALL_PREFIX}/${OGRE_MEDIA_PATH}")
```

Comment:
this changes the behavior compared to the current state, which will break things.

The difference between `Media`, `./Media` and `../Media` is important